types/libhoney/index.d.ts

```diff
@@ -0,0 +1,115 @@
+// Type definitions for libhoney 4.1
+// Project: https://github.com/honeycombio/libhoney-js#readme
+// Definitions by: dcposch <https://github.com/dcposch>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = libhoney;
+
+declare class libhoney {
```

Comment:
the symbol is named `Libhoney`:
https://github.com/honeycombio/libhoney-js/blob/main/src/libhoney.js#L74

Reply:
Good call, fixed.